app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <% if Company.find_by(name: '3sixD').logo.present? %>
+      <% @default_company = Company.find_by(name: '3sixD') || Company.find_by(name: '3sixd') %>
```

Comment:
@leonardk1 Instead of having two `if statements`  the short hand to do that is just use the OR operator and it works fine, whether the name is 3sixd or 3sixD, that query returns what I need